config/schedule.rb

```diff
@@ -35,6 +35,18 @@ def local(time)
   runner "PatientDeduplication::Runner.new(PatientDeduplication::Strategies.identifier_and_full_name_match).perform"
 end
 
+every :day, at: local("02:30 am").utc, roles: [:cron] do
+  if Flipper.enabled?(:live_experiment)
+    runner "ExperimentControlService.start_medication_reminder_experiment('name')" # needs the real experiment name
+  end
+end
+
+every :day, at: local("03:00 am").utc, roles: [:cron] do
+  if Flipper.enabled?(:live_experiment)
+    runner "AppointmentNotification::ScheduleExperimentReminders.perform_later"
```

Comment:
I assume perform_later will work here. None of the other scheduled jobs directly use sidekiq. I'm not sure if there's a reason for that.

Reply:
I think we have been using ActiveJob where possible, so `perform_later` seems fine to me.